csunplugged/utils/custom_converter_templates/image-inline.html

```diff
@@ -0,0 +1,22 @@
+<span class="d-inline">
+<img src="{{ file_path }}"
+class="inline-image"
+{% if alt %} alt="{{ alt }}"{% endif %}
+{% if hover_text %} title="{{ hover_text }}"{% endif %}>
+{% if caption %}
+<p class="text-center">
+{% if caption_link %}
+<a href="{{ caption_link }}">
+{% endif %}
+{{ caption }}
+{% if caption_link %}
+</a>
+{% endif %}
+</p>
+{% endif %}
+{% if source_link %}
+<p class="text-center">
+<a href="{{ source_link }}">Source</a>
+</p>
+{% endif %}
+</span>
```

Comment:
All Verto HTML files have no indentation for testing purposes, so when copying the base template I've kept that style. That isn't required though, so I can indent this.